src/hotspot/share/logging/logAsyncWriter.cpp

```diff
@@ -149,6 +150,10 @@ void AsyncLogWriter::write() {
 
   if (req > 0) {
     assert(req == 1, "AsyncLogWriter::flush() is NOT MT-safe!");
+    // LogFileOutput::write_block() has called fflush().
```

Comment:
write_block or write_blocking?

Reply:
delete this. 
now both LFO and LFSO do fflush() in write_blocking(), we don't need the special treatment here.